test/custom.css

```diff
@@ -12,6 +12,11 @@ body {
   --skipped-color:#770077;
 }
 
+/* Hide darkmode toggle button */
+.fa-toggle-off {
```

Comment:
Hi @BaCaRoZzo 

Not sure why you want to hide the button, can you explain the use case? Hiding this one would prevent toggling back

Reply:
@wswebcreation when a custom theming is applied the toggle may or may not work, depending on the theming. In my case it didn't work so I hidden for myself.

I thought providing the user with a quick way to hide it was valuable as well and thus I've added it. Ofc I can remove it, no biggie. The important bit is the fix, indeed. :slightly_smiling_face:

Reply:
I agree, hiding toggle-off in custom.css is good one since it doesn't required to show it as it is custom anyway.

Reply:
Thanks for explaining!